data/json/items/melee/bludgeons.json

```diff
@@ -905,20 +905,20 @@
     "type": "GENERIC",
     "id": "aztec_sword_stone",
     "name": { "str": "stone lined macuahuitl" },
-    "description": "An improvised version of the close combat weapon that was mainly used by Aztecs.  It's not very sturdy, but it will break someone's skull open and give them a good cut.",
+    "description": "A decent enough replica of the close combat weapon that was mainly used by Aztecs.  It has carefully embedded stone blades in place of the original obsidian.  Its capable of breaking someone's skull open and give them a good cut.",
     "weight": "2050 g",
     "volume": "3800 ml",
-    "price": "10 USD",
+    "price": "20 USD",
     "price_postapoc": "2 USD 50 cent",
     "symbol": "/",
     "color": "brown",
     "material": [ "wood", "stone" ],
     "longest_side": "100 cm",
-    "flags": [ "NONCONDUCTIVE", "FRAGILE_MELEE" ],
+    "flags": [ "NONCONDUCTIVE" ],
```

Comment:
Slashing at someone with glued-in-place stones is fragile melee no matter how you look at it.  This thing is not staying in one piece indefinitely.  If you want to look into some kind of intermediate level of fragility where it lasts longer sure thing, but we're not jumping all the way to the default level of durability with anything macuahuitl shaped.